flattentool/input.py

```diff
@@ -246,10 +260,11 @@ def do_unflatten(self):
                 root_id_or_none = line.get(self.root_id) if self.root_id else None
                 cells = OrderedDict()
                 for k, header in enumerate(line):
-                    if actual_headings:
-                        cells[header] = Cell(line[header], (sheet_name, _get_column_letter(k+1), j+2, actual_headings[k]))
+                    headings = actual_headings[k] if actual_headings else header
+                    if self.vertical_orientation:
+                        cells[header] = Cell(line[header], (sheet_name, str(k+1), j+2, headings))
```

Comment:
No, because this would lead to broken "Rows" output in Cove. We think there's also problems with this/the current approach, but there's a new issue for this https://github.com/OpenDataServices/flatten-tool/issues/153